swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = None, age = None):
```

Comment:
A `category` parameter here makes it possible for someone to create an instance of `Clothing` and pass in a different value.  Removing `category` from the parameters into the function body would ensure that an instance of class `Clothing` always has the category "Clothing":
```suggestion
    def __init__(self, condition = None, age = None):
        category = "Clothing"
```